test/wallet/signer.ts

```diff
@@ -0,0 +1,303 @@
+      },
+    },
+  ],
+  SigningPubKey: '',
+  TransactionType: 'TrustSet',
+}
+
+const multisignTxToCombine2: Transaction = {
+  Account: 'rEuLyBCvcw4CFmzv8RepSiAoNgF8tTGJQC',
+  Fee: '30000',
+  Flags: 262144,
+  LimitAmount: {
+    currency: 'USD',
+    issuer: 'rHb9CJAWyB4rj91VRWn96DkukG4bwdtyTh',
+    value: '100',
+  },
+  Sequence: 2,
+  Signers: [
+    {
+      Signer: {
+        Account: 'rJvuSQhQR37czfxRou4vNWaM97uEhT4ShE',
+        SigningPubKey:
+          '02B78EEA571B2633180834CC6E7B4ED84FBF6811D12ECB59410E0C92D13B7726F5',
+        TxnSignature:
+          '304502210098009CEFA61EE9843BB7FC29B78CFFAACF28352A4A7CF3AAE79EF12D79BA50910220684F116266E5E4519A7A33F7421631EB8494082BE51A8B03FECCB3E59F77154A',
+      },
+    },
+  ],
+  SigningPubKey: '',
+  TransactionType: 'TrustSet',
+}
+
+const expectedMultisign: Transaction = {
+  Account: 'rEuLyBCvcw4CFmzv8RepSiAoNgF8tTGJQC',
+  Fee: '30000',
+  Flags: 262144,
+  LimitAmount: {
+    currency: 'USD',
+    issuer: 'rHb9CJAWyB4rj91VRWn96DkukG4bwdtyTh',
+    value: '100',
+  },
+  Sequence: 2,
+  Signers: [
+    {
+      Signer: {
+        Account: 'rsA2LpzuawewSBQXkiju3YQTMzW13pAAdW',
+        SigningPubKey:
+          '02B3EC4E5DD96029A647CFA20DA07FE1F85296505552CCAC114087E66B46BD77DF',
+        TxnSignature:
+          '30450221009C195DBBF7967E223D8626CA19CF02073667F2B22E206727BFE848FF42BEAC8A022048C323B0BED19A988BDBEFA974B6DE8AA9DCAE250AA82BBD1221787032A864E5',
+      },
+    },
+    {
+      Signer: {
+        Account: 'rJvuSQhQR37czfxRou4vNWaM97uEhT4ShE',
+        SigningPubKey:
+          '02B78EEA571B2633180834CC6E7B4ED84FBF6811D12ECB59410E0C92D13B7726F5',
+        TxnSignature:
+          '304502210098009CEFA61EE9843BB7FC29B78CFFAACF28352A4A7CF3AAE79EF12D79BA50910220684F116266E5E4519A7A33F7421631EB8494082BE51A8B03FECCB3E59F77154A',
+      },
+    },
+  ],
+  SigningPubKey: '',
+  TransactionType: 'TrustSet',
+}
+
+describe('Signer', function () {
+  it('sign', function () {
+    // Test case data generated using this tutorial - https://xrpl.org/send-xrp.html#send-xrp
+    const tx3: Transaction = {
+      TransactionType: 'Payment',
+      Account: 'rHLEki8gPUMnF72JnuALvnAMRhRemzhRke',
+      Amount: '22000000',
+      Destination: 'rPT1Sjq2YGrBMTttX4GZHjKu9dyfzbpAYe',
+      Flags: 2147483648,
+      LastLedgerSequence: 20582339,
+      Fee: '12',
+      Sequence: 20582260,
+    }
+    const signedTxBlob =
+      '120000228000000024013A0F74201B013A0FC36140000000014FB18068400000000000000C732102A8A44DB3D4C73EEEE11DFE54D2029103B776AA8A8D293A91D645977C9DF5F544744730450221009ECB5324717E14DD6970126271F05BC2626D2A8FA9F3797555D417F8257C1E6002206BDD74A0F30425F2BA9DB69C90F21B3E27735C190FB4F3A640F066ACBBF06AD98114B3263BD0A9BF9DFDBBBBD07F536355FF477BF0E98314F667B0CA50CC7709A220B0561B85E53A48461FA8'
+
+    const wallet = Wallet.fromSeed(seed)
```

Comment:
You can have this be a global variable, instead of running this in every test - the wallet setup won't affect anything.